string base64
string content_type
string name